### ISSUE TITLE
Error when, /wild.....

### ISSUE THREAD

**Issue by BaconOFBurger** [poggit-orphanage/essentialsTP#18]
Here logs:
[06:50:33] [Server thread/CRITICAL]: Unhandled exception executing command 'wild' in wild: Call to undefined method pocketmine\level\Level::generateChunk()
[06:50:33] [Server thread/CRITICAL]: Error: "Call to undefined method pocketmine\level\Level::generateChunk()" (EXCEPTION) in "essentialsTP-master/src/essentialsTP/essentialsTP" at line 1052
[06:50:33] [Server thread/DEBUG]: #0 src/pocketmine/command/PluginCommand(58): essentialsTP\essentialsTP->onCommand(pocketmine\Player object, pocketmine\command\PluginCommand object, string wild, array Array())
[06:50:33] [Server thread/DEBUG]: #1 src/pocketmine/command/SimpleCommandMap(258): pocketmine\command\PluginCommand->execute(pocketmine\Player object, string wild, array Array())
[06:50:33] [Server thread/DEBUG]: #2 src/pocketmine/Server(1946): pocketmine\command\SimpleCommandMap->dispatch(pocketmine\Player object, string wild)
[06:50:33] [Server thread/DEBUG]: #3 src/pocketmine/Player(2188): pocketmine\Server->dispatchCommand(pocketmine\Player object, string wild)
[06:50:33] [Server thread/DEBUG]: #4 src/pocketmine/network/mcpe/PlayerNetworkSessionAdapter(222): pocketmine\Player->chat(string /wild)
[06:50:33] [Server thread/DEBUG]: #5 src/pocketmine/network/mcpe/protocol/CommandRequestPacket(54): pocketmine\network\mcpe\PlayerNetworkSessionAdapter->handleCommandRequest(pocketmine\network\mcpe\protocol\CommandRequestPacket object)
[06:50:33] [Server thread/DEBUG]: #6 src/pocketmine/network/mcpe/PlayerNetworkSessionAdapter(92): pocketmine\network\mcpe\protocol\CommandRequestPacket->handle(pocketmine\network\mcpe\PlayerNetworkSessionAdapter object)
[06:50:33] [Server thread/DEBUG]: #7 src/pocketmine/network/mcpe/protocol/BatchPacket(114): pocketmine\network\mcpe\PlayerNetworkSessionAdapter->handleDataPacket(pocketmine\network\mcpe\protocol\CommandRequestPacket object)
[06:50:33] [Server thread/DEBUG]: #8 src/pocketmine/network/mcpe/PlayerNetworkSessionAdapter(92): pocketmine\network\mcpe\protocol\BatchPacket->handle(pocketmine\network\mcpe\PlayerNetworkSessionAdapter object)
[06:50:33] [Server thread/DEBUG]: #9 src/pocketmine/Player(3031): pocketmine\network\mcpe\PlayerNetworkSessionAdapter->handleDataPacket(pocketmine\network\mcpe\protocol\BatchPacket object)
[06:50:33] [Server thread/DEBUG]: #10 src/pocketmine/network/mcpe/RakLibInterface(153): pocketmine\Player->handleDataPacket(pocketmine\network\mcpe\protocol\BatchPacket object)
[06:50:33] [Server thread/DEBUG]: #11 vendor/pocketmine/raklib/server/ServerHandler(98): pocketmine\network\mcpe\RakLibInterface->handleEncapsulated(string 192.168.0.23 64158, raklib\protocol\EncapsulatedPacket object, integer 0)
[06:50:33] [Server thread/DEBUG]: #12 src/pocketmine/network/mcpe/RakLibInterface(96): raklib\server\ServerHandler->handlePacket()
[06:50:33] [Server thread/DEBUG]: #13 src/pocketmine/network/Network(89): pocketmine\network\mcpe\RakLibInterface->process()
[06:50:33] [Server thread/DEBUG]: #14 src/pocketmine/Server(2495): pocketmine\network\Network->processInterfaces()
[06:50:33] [Server thread/DEBUG]: #15 src/pocketmine/Server(2243): pocketmine\Server->tick()
[06:50:33] [Server thread/DEBUG]: #16 src/pocketmine/Server(2119): pocketmine\Server->tickProcessor()
[06:50:33] [Server thread/DEBUG]: #17 src/pocketmine/Server(1701): pocketmine\Server->start()
[06:50:33] [Server thread/DEBUG]: #18 src/pocketmine/PocketMine(305): pocketmine\Server->__construct(BaseClassLoader object, pocketmine\utils\MainLogger object, string C:\Users\Guest\Desktop\BrickLine ALPHA-2.3\, string C:\Users\Guest\Desktop\BrickLine ALPHA-2.3\plugins\)
[06:50:33] [Server thread/DEBUG]: #19 C:/Users/Guest/Desktop/BrickLine ALPHA-2.3/PocketMine-MP.phar(1): require(string phar://C:/Users/Guest/Desktop/BrickLine ALPHA-2.3/PocketMine-MP.phar/src/pocketmine/PocketMine.php)
Please fix it or help me.

**Comment by Haxley** [comment]
Same as #17

In the most recent server software release they, “Removed dead generator things like `Level->generateChunk()`, `Level->regenerateChunk()`, `GenerationTask,`” which /wild used.

**Comment by teracube** [comment]
same issue with
PocketMine-MP version 1.7dev-963
Essentials TP dev 26 + change ALPHA12

```
2018-05-07 [23:04:23] [Server thread/CRITICAL]: Unhandled exception executing command 'wild' in wild: Call to undefined method pocketmine\level\Level::generateChunk()
2018-05-07 [23:04:23] [Server thread/CRITICAL]: Error: "Call to undefined method pocketmine\level\Level::generateChunk()" (EXCEPTION) in "essentialsTP_dev-26.phar/src/essentialsTP/essentialsTP" at line 1052
2018-05-07 [23:04:23] [Server thread/DEBUG]: #0 src/pocketmine/command/PluginCommand(58): essentialsTP\essentialsTP->onCommand(pocketmine\Player object, pocketmine\command\PluginCommand object, string wild, array Array())
2018-05-07 [23:04:23] [Server thread/DEBUG]: #1 src/pocketmine/command/SimpleCommandMap(258): pocketmine\command\PluginCommand->execute(pocketmine\Player object, string wild, array Array())
2018-05-07 [23:04:23] [Server thread/DEBUG]: #2 src/pocketmine/Server(1946): pocketmine\command\SimpleCommandMap->dispatch(pocketmine\Player object, string wild)
2018-05-07 [23:04:23] [Server thread/DEBUG]: #3 src/pocketmine/Player(2189): pocketmine\Server->dispatchCommand(pocketmine\Player object, string wild)
2018-05-07 [23:04:23] [Server thread/DEBUG]: #4 src/pocketmine/network/mcpe/PlayerNetworkSessionAdapter(222): pocketmine\Player->chat(string /wild)
2018-05-07 [23:04:23] [Server thread/DEBUG]: #5 src/pocketmine/network/mcpe/protocol/CommandRequestPacket(54): pocketmine\network\mcpe\PlayerNetworkSessionAdapter->handleCommandRequest(pocketmine\network\mcpe\protocol\CommandRequestPacket object)
2018-05-07 [23:04:23] [Server thread/DEBUG]: #6 src/pocketmine/network/mcpe/PlayerNetworkSessionAdapter(92): pocketmine\network\mcpe\protocol\CommandRequestPacket->handle(pocketmine\network\mcpe\PlayerNetworkSessionAdapter object)
2018-05-07 [23:04:23] [Server thread/DEBUG]: #7 src/pocketmine/network/mcpe/protocol/BatchPacket(114): pocketmine\network\mcpe\PlayerNetworkSessionAdapter->handleDataPacket(pocketmine\network\mcpe\protocol\CommandRequestPacket object)
2018-05-07 [23:04:23] [Server thread/DEBUG]: #8 src/pocketmine/network/mcpe/PlayerNetworkSessionAdapter(92): pocketmine\network\mcpe\protocol\BatchPacket->handle(pocketmine\network\mcpe\PlayerNetworkSessionAdapter object)
2018-05-07 [23:04:23] [Server thread/DEBUG]: #9 src/pocketmine/Player(3032): pocketmine\network\mcpe\PlayerNetworkSessionAdapter->handleDataPacket(pocketmine\network\mcpe\protocol\BatchPacket object)
2018-05-07 [23:04:23] [Server thread/DEBUG]: #10 src/pocketmine/network/mcpe/RakLibInterface(146): pocketmine\Player->handleDataPacket(pocketmine\network\mcpe\protocol\BatchPacket object)
2018-05-07 [23:04:23] [Server thread/DEBUG]: #11 vendor/pocketmine/raklib/server/ServerHandler(98): pocketmine\network\mcpe\RakLibInterface->handleEncapsulated(string 90.xx.xx.61 58045, raklib\protocol\EncapsulatedPacket object, integer 0)
2018-05-07 [23:04:23] [Server thread/DEBUG]: #12 src/pocketmine/network/mcpe/RakLibInterface(93): raklib\server\ServerHandler->handlePacket()
2018-05-07 [23:04:23] [Server thread/DEBUG]: #13 src/pocketmine/network/Network(89): pocketmine\network\mcpe\RakLibInterface->process()
2018-05-07 [23:04:23] [Server thread/DEBUG]: #14 src/pocketmine/Server(2495): pocketmine\network\Network->processInterfaces()
2018-05-07 [23:04:23] [Server thread/DEBUG]: #15 src/pocketmine/Server(2243): pocketmine\Server->tick()
2018-05-07 [23:04:23] [Server thread/DEBUG]: #16 src/pocketmine/Server(2119): pocketmine\Server->tickProcessor()
2018-05-07 [23:04:23] [Server thread/DEBUG]: #17 src/pocketmine/Server(1701): pocketmine\Server->start()
2018-05-07 [23:04:23] [Server thread/DEBUG]: #18 src/pocketmine/PocketMine(305): pocketmine\Server->__construct(BaseClassLoader object, pocketmine\utils\MainLogger object, string /home/gsp_1485/g3.hmserv.eu-19144/, string /home/gsp_1485/g3.hmserv.eu-19144/plugins/)
2018-05-07 [23:04:23] [Server thread/DEBUG]: #19 /home/gsp_1485/g3.hmserv.eu-19144/PocketMine-MP.phar(1): require(string phar:///home/gsp_1485/g3.hmserv.eu-19144/PocketMine-MP.phar/src/pocketmine/PocketMine.php)
```

**Comment by Awzaw** [comment]
Fixed in 37c331e0d27b
